lib/heuristics/concerns/scheduling_end_phase.rb

```diff
@@ -239,7 +239,8 @@ def empty_underfilled
 
           all_empty = false
 
-          next if route_data[:stops].sum{ |stop| @services_data[stop[:id]][:raw].exclusion_cost || 0 } >= route_data[:cost_fixed]
+          next if route_data[:stops].sum{ |stop| @services_data[stop[:id]][:raw].exclusion_cost || 0 } >=
+                  route_data[:vehicle].cost_fixed
```

Comment:
this looks weird, rubocop should have marked it.

in any case it is better to keep the conditional operator (==, >=, etc) at the same line with part of the left hand side and right hand side at the same time (to clearly see what gets compared with what) like
```suggestion
          next if route_data[:stops].sum{ |stop| 
                    @services_data[stop[:id]][:raw].exclusion_cost || 0
                  } >= route_data[:vehicle].cost_fixed
```
otherwise, t looks like a separate condition on its own.

(`|| 0` can be replaced with `.to_i` or `.to_f` depending on if exclusion_cost is a float or int)